tasmota/language/it-IT.h

```diff
@@ -120,55 +120,55 @@
 #define D_MULTI_PRESS "multi-pressione"
 #define D_NOISE "Rumore"
 #define D_NONE "Nessuno"
-#define D_OFF "Off"
+#define D_OFF "OFF"
 #define D_OFFLINE "Offline"
-#define D_OK "Ok"
-#define D_ON "On"
+#define D_OK "OK"
+#define D_ON "ON"
 #define D_ONLINE "Online"
 #define D_PASSWORD "Password"
 #define D_PORT "Porta"
 #define D_POWER_FACTOR "Fattore di potenza"
 #define D_POWERUSAGE "Potenza"
-#define D_POWERUSAGE_ACTIVE "Potenza Attiva"
-#define D_POWERUSAGE_APPARENT "Potenza Apparente"
-#define D_POWERUSAGE_REACTIVE "Potenza Reattiva"
+#define D_POWERUSAGE_ACTIVE "Potenza attiva"
```

Comment:
Anche se grammaticalmente corretto usare la minuscola, noi usiamo la maiuscola per tutte le parole. Verifica nelle altre traduzioni. Questo vale per tutte le altre stringhe che hai cambiato.

Reply:
E usecondo me (e anche Adobe/Microsoft/etc) una convenzione sbagliata.

I software in lingua italiana (es. Adobe/Micorosft) non usano per l'italiano tutte lettere maiuscole ma sola la prima lettera maiuscola (ad eccezioend ei nomi propri).

Le lettere tutte maiuscole sono tipiche della lingua inglese. Non italiana.

Reply:
"Anche se grammaticalmente corretto usare la minuscola", come ti ho scritto prima, significa che so come si scrive in Italiano.  
Quello che ti sto dicendo é che noi preferiamo le maiuscole, da una occhiata nelle altre traduzioni. 
Per adesso lascia le stringhe così.

Reply:
Per favore allora non chiamarla traduzione italiana perché questo NON è una traduzione italiana.

L'italiano ha le sue regole e se vuoi fare una traduzione italiana usi le regole della grammatica 
 italiana altrimenti è una traduzione custom ma non si definire italiana.

Queste sono regole basi delle traduzioni del software.
Le sua Microsoft le sua Adobe che sono giganti del software che conoscono e usano l'italiano in modo coretto (loro) 

In questa traduzione attuale si sono inoltre usate una serie di convenzioni dove si è semplicemente tradotto dall'inglese senza rispettare il flusso discorso italiano

web server -> server web (etc)-

Reply:
Non essendoci il soggetto (stringhe originali / stringhe modificate) tu intendevi una cosa e io avevo capito un'altra.
Meglio così.